app/src/main/kotlin/com/wire/android/ui/common/WireDropDown.kt

```diff
@@ -0,0 +1,157 @@
+package com.wire.android.ui.common
+
+import CheckIcon
```

Comment:
I think we need to add a proper package to `MutingOptionsSheetContent` (it's where this `CheckIcon` comes from) because it's not right like this 😄